express-delete/index.js

```diff
@@ -0,0 +1,44 @@
+const express = require('express');
+
+const app = express();
+
+const grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92
+  }
+};
+
+const gradesArr = [];
+
+app.get('/api/grades', (req, res) => {
+  for (const key in grades) {
+    gradesArr.push(grades[key]);
+  }
+  res.json(gradesArr);
+});
+
+app.listen(3000);
+
+app.delete('/api/grades/:id', (req, res) => {
+  deleteGrade(req.params.id);
+  res.sendStatus(204);
```

Comment:
You can delete from an object using the `delete` operator.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/delete